docs/modules/ROOT/examples/dds/map/Employee.java

```diff
@@ -1,19 +1,14 @@
-import java.io.Serializable;
-
 //tag::emp[]
-public class Employee implements Serializable {
+public class Employee {
 
     private final String surname;
 
     public Employee(String surname) {
         this.surname = surname;
     }
 
-    @Override
-    public String toString() {
-        return "Employee{" +
-                "surname='" + surname + '\'' +
-                '}';
+    public HazelcastJsonValue toHazelcastJsonValue() {
+        return new HazelcastJsonValue("{ \"surname\": \"" + surname + "\" }");
```

Comment:
I think this can just be a static method and that way we can eliminate one unnecessary object creation while adding some entries to map. I guess we don't show this step, but if we change the text around this class, I think we can just say we are going to use HJV instead of a class

Reply:
This method uses surname field of Employee. How can it be static?

Reply:
Updated in b4807f58c20f5ef99d3cf2e9bdb595aaa24c69f9